reference/conanfile/tools/cmake/cmaketoolchain.rst

```diff
@@ -366,6 +366,7 @@ conf
 - ``tools.build:sharedlinkflags`` list of extra linker flags that will be appended to ``CMAKE_SHARED_LINKER_FLAGS_INIT``.
 - ``tools.build:exelinkflags`` list of extra linker flags that will be appended to ``CMAKE_EXE_LINKER_FLAGS_INIT``.
 - ``tools.build:defines`` list of preprocessor definitions that will be used by ``add_definitions()``.
+- ``tools.build:linker_scripts`` list of linker scripts, each of which will be prepended with ``-T`` and appended to ``CMAKE_EXE_LINKER_FLAGS_INIT``.
```

Comment:
I would suggest adding a further clarification that this is to be used when the linker understands the `-T` option and can parse linker scripts. I know it should be implied if someone uses this, but won't hurt, since on other platforms the native linker does not operate with linker scripts.

Reply:
@jcar87 I was thinking the same thing, but I didn't want to put it in the `CMakeToolchain` stuff only, since it may apply to Meson and Autotools as well. Should I describe this caveat in all three places?

Reply:
Yes, please do :)

Reply:
Done.